File: src/cli/tests/fixtures/bootstrap/compose/src/PrimaryButton.kt
package com.legacy.ui

import androidx.compose.runtime.Composable
import androidx.compose.ui.tooling.preview.Preview

@Composable
fun PrimaryButton(appearance: String, disabled: Boolean, onClick: () -> Unit) {
    // background #1a73e8
}

@Preview
@Composable
fun PrimaryButtonPreview() {
    PrimaryButton(appearance = "primary", disabled = false, onClick = {})
}

File: src/cli/tests/fixtures/bootstrap/swiftui/Sources/PrimaryButton.swift
import SwiftUI

struct PrimaryButton: View {
    var appearance: String
    var disabled: Bool
    var onClick: () -> Void

    var body: some View {
        Button(action: onClick) {
            Text("Save")
        }
        .background(Color(red: 0.102, green: 0.451, blue: 0.91)) // #1a73e8
    }
}

#Preview {
    PrimaryButton(appearance: "primary", disabled: false, onClick: {})
}
